#pophistory assembly and analyses for whole genomes

##Part 1: map raw reads, call SNPs, create grouped genotyped vcfs 

##Part 2: filter vcfs & convert to inputs for analyses 

##Parts 3: Run NGSadmix, plot outputs in R, compare log likelihoods across runs

##Part 4: Create a PCA from the filtered SNP data in R, calculate and plot the variance explained

##Part 5: Run Treemix, plot outputs in R, perform ad hoc test for number of migration events 

##Part 6: Run PLINK ROH on VCF, plot outputs in R
